fileblock/src/emulator.rs

```diff
@@ -53,42 +53,34 @@ impl BlockStorage for FileBlockEmulator {
                 "buffer does not contain enough space to read block",
             ));
         }
-        // FIXME(allancalix): Keep a seek pointer in file descriptor to avoid
```

Comment:
how would this work? just out of curiousity

Reply:
It actually doesn’t matter from what I understand. Whether you seek from start or from some file pointer the disk read is the same (since it’ll point to the same position).

Linux will keep a pointer to the current seek location, that’s how consecutive reads don’t just read the first block over and over.